cachito/__init__.py

```diff
@@ -0,0 +1,19 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+import os
+from flask import Flask
+
+app = Flask(__name__)
```

Comment:
Let's refactor this to use an application factory. That can be in a separate PR.

Reply:
Awesome, thanks for suggesting that. This is done now.